keyboards/clueboard/60/rules.mk

```diff
@@ -1,5 +1,5 @@
 # MCU name
-MCU = STM32F303
+MCU = STM32F303xB
```

Comment:
There are two PCB's that are otherwise identical, one with an xC and one with an xB. Given supply chain issues for the xC and xB I think we'll see this with other boards in the future.

Reply:
That's ... a mess.  

Would it be better to assume the smaller amount, even if that isn't accurate for every board?

Reply:
On some MCUs, the smaller amount is quite small (64KB). QMK barely fits into that space, and some options have to be turned off to get there.

Reply:
Probably exacerbated now with @awkannan using APM32's due to supply chain issues of STM32's.